src/main/java/org/csu/jpetstore/service/impl/AccountServiceImpl.java

```diff
@@ -13,6 +13,50 @@ public Account getAccount(String username) {
         accountDAO = new AccountDAOImpl();
         return accountDAO.getAccountByUsername(username);
     }
+    
+    package org.csu.jpetstore.service.impl;
+import org.csu.jpetstore.domain.Account;
+import org.csu.jpetstore.persistence.AccountDAO;
+import org.csu.jpetstore.persistence.impl.AccountDAOImpl;
+import org.csu.jpetstore.service.AccountService;
+
+public class AccountServiceImpl implements AccountService {
+
+    private AccountDAO accountDAO;
+
+    @Override
+    public Account getAccount(String username) {
```

Comment:
The code is logical and compact but with few problems remained. For example, there are still rooms for improve in the skillfulness of the code.